### PR TITLE
doc: added documentation for input type=file

### PR DESCRIPTION
**Pull request by el1f**
Added some explaination on how to implement this form type in React as suggested in #86

### CODE REVIEW

File: content/docs/forms.md [hunk 1]
```diff
@@ -190,6 +190,61 @@ Overall, this makes it so that `<input type="text">`, `<textarea>`, and `<select
...
+
+    this.handleFileChange = this.handleFileChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleFileChange(e) {
+    e.preventDefault();
+
+    this.setState({
+      file: e.target.files[0]
```
Comment (tsriram) [inline content/docs/forms.md]: Don't think the file needs to be saved in state, unless I'm overlooking something. Probably having a `ref` to the file input and use it to access the file(s) in submit handler (or wherever required) would be a better option IMHO. See if this makes sense - https://codepen.io/tsriram/pen/qVNoZL?editors=0011

Comment (el1f) [reply content/docs/forms.md]: Yeah, using the file by ref is definitly a betetr solution. My bad. Will change it ASAP

Comment (bvaughn) [reply content/docs/forms.md]: Friendly ping 😄

Comment (el1f) [reply content/docs/forms.md]: Sorry guys, been away for a while and didn't notice the progress of the issue. Seems like @vishalvrv9 has worked on a fix of my PR based on @tsriram's suggestion. Should I go ahead and fix mine or would it better to merge his?

Comment (el1f) [reply content/docs/forms.md]: Yeah, had some issues so I couldn't track down my PR after submitting it but the man definitly deserves some attention for his efforts ;)

**Pull request closed without merging**